doc/refexpr.rst

```diff
@@ -129,3 +129,47 @@ references in this structure, e.g.::
       myCons[i] = @addConstraint(m, x[i] >= i)
     end
 
+Vectorized operations
+^^^^^^^^^^^^^^^^^^^^^
+
+JuMP supports vectorized expressions and constraints for linear and quadratic models. Although this syntax may
+be familiar for users coming from other modeling languages, we caution that this syntax can be slow---especially
+for large operations---and that it often can destroy the sparse, algebraic structure that makes modeling
```

Comment:
What does "destroy" mean here? If you use sparse matrices, it shouldn't destroy the structure.